samples/kubernetes/end2end/docs/03-wls-operator.md

```diff
@@ -12,6 +12,7 @@ We need two images, the WebLogic Server Kubernetes Operator image from [Docker H
 Pull the WebLogic Sever Kubernetes Operator 2.3.0 image.
 ```
 docker login
+for logging into oracle registry use with sso creds use:  docker login container-registry.oracle.com
```

Comment:
Each of these lines is executable, and this additional line is not.  This is not the correct location for this comment.